$OpenBSD: patch-rights,v 1.3 2001/01/09 15:25:26 camield Exp $

This port wants to change the rights on PREFIX/ and PREFIX/bin to 02755.
Usually PREFIX is /usr/local. The rights on /usr/local and /usr/local/bin
should be 0755 according to /etc/mtree/4.4BSD.dist.

Also, use PREFIX/share/djbdns instead of /etc for dnsroots.global.

--- hier.c.orig	Sun Jan  7 06:08:55 2001
+++ hier.c	Mon Jan  8 10:46:46 2001
@@ -2,10 +2,12 @@
 
 void hier()
 {
-  c("/","etc","dnsroots.global",-1,-1,0644);
+  h(auto_home,-1,-1,0755);
+  d(auto_home,"bin",-1,-1,0755);
 
-  h(auto_home,-1,-1,02755);
-  d(auto_home,"bin",-1,-1,02755);
+  d(auto_home,"share",-1,-1,0755);
+  d(auto_home,"share/djbdns",-1,-1,0755);
+  c(auto_home,"share/djbdns","dnsroots.global",-1,-1,0644);
 
   c(auto_home,"bin","dnscache-conf",-1,-1,0755);
   c(auto_home,"bin","tinydns-conf",-1,-1,0755);
